octopus-agent/src/main/java/octopus/teamcity/agent/OctopusBuildInformationBuildProcess.java

```diff
@@ -40,146 +40,145 @@
 
 public class OctopusBuildInformationBuildProcess extends OctopusBuildProcess {
 
-  private final File checkoutDir;
```

Comment:
It looks like this code reformat causes the build to fail - https://github.com/OctopusDeploy/Octopus-TeamCity/runs/5655504829?check_suite_focus=true#step:4:57

Reply:
Fixed the editor settings and corrected the formatting.